delta/rdf/src/test/scala/ch/epfl/bluebrain/nexus/delta/rdf/jsonld/ExpandedJsonLdSpec.scala

```diff
@@ -153,7 +153,7 @@ class ExpandedJsonLdSpec extends CatsEffectSpec with Fixtures with GraphHelpers
 
       val input =
         json"""{"@context": ["http://localhost/c", {"a": "http://localhost/a"} ], "a": "A", "c": "C", "d": "D"}"""
-      ExpandedJsonLd(input).rejected shouldEqual RemoteContextCircularDependency(iri"http://localhost/c")
+      ExpandedJsonLd(input).rejectedWith[ConversionError].getMessage contains "Too many contexts"
```

Comment:
With Json-LD 1.1 and imported contexts, the circular dependency check is replaced by a maximum number of loaded contexts